mk/config.mk

```diff
@@ -1038,3 +1038,7 @@ CFG_TA_OPTEE_CORE_API_COMPAT_1_1 ?= n
 #              - PerInstance/AttestationTest#
 # Note that this violates GP requirements of HMAC size range.
 CFG_HMAC_64_1024_RANGE ?= n
+
+#Enable a hardware pbkdf2 function
```

Comment:
Please add a space after `#`, the same on the line below.

Reply:
Modifications are made.